pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -171,10 +198,22 @@ func NewElasticsearchKubeControllers(cfg *KubeControllersConfiguration) *kubeCon
 		kubeControllerAllowTigeraPolicy = esKubeControllersAllowTigeraPolicy(cfg)
 	}
 
-	enabledControllers := []string{"authorization", "elasticsearchconfiguration"}
+	var enabledControllers []string
+
+	if !cfg.Tenant.MultiTenant() {
+		enabledControllers = append(enabledControllers, "authorization", "elasticsearchconfiguration")
+		if cfg.ManagementCluster != nil {
+			enabledControllers = append(enabledControllers, "managedcluster")
+		}
+	} else {
+		if cfg.ManagementCluster != nil {
```

Comment:
Could be else-if to simplify the code. But if cfg.Tenant.MultiTenant()=True, doesn't that always mean that cfg.ManagementCluster != nil ?